syntax = "proto2";
message userAcc {
  required int32 id = 1;
  required string name = 2;
  optional string displayName = 3;
  required string pwdhash = 4;

  repeated string messages = 5;

  optional string publicIk = 6;
  optional string privateIk = 7;
  optional string publicPk = 8;
  optional string privatePk = 9;

  optional string signedPk = 10;

  repeated string oneTimePublic = 11;
  repeated string oneTimePrivate = 12;
}
